install/linux/docker-ee/suse.md

```diff
@@ -164,11 +164,14 @@ Before you install Docker EE for the first time on a new host machine, you need
 to set up the Docker repository. Afterward, you can install and update Docker EE
 from the repository.
 
-There are currently two versions of Docker EE Engine available:
+The current version of Docker EE Engine is 18.09 for using with Docker Enterprise Edition 2.1.
 
-* 18.03 - Use this version if you're only running Docker EE Engine.
-* 17.06 - Use this version if you're using Docker Enterprise Edition 2.0 (Docker
-Engine, UCP, and DTR).
+> ***NOTE:*** There are two older versions of Docker EE Engine available:
+> * 18.03 - Use this version if you're only running Docker EE Engine.
```

Comment:
Same here. Whether we specify `stable-18.03` or just the version number is something I'll leave to the engineers to make a call on. But keep it consistent for the notes on `suse` and `ubuntu`.

Reply:
I think `18.03` should be fine; the _version_ is `18.03`, but the "fixed" repositories happen to have `stable-` as a prefix. I think it's best to only mention the `stable-` prefix where it's needed (so in the command-line examples used for doing the actual install)

Hm... reading this again, I wonder if we should slightly rephrase this. Currently it may be interpreted as "If you're only running Docker EE Engine, don't install 18.09, but install 18.03 instead" (which is obviously not what we want)

Reply:
@bermudezmt Can you let me know where specifically I need to change stable? @thaJeztah said the scripts and configuration should stay the same, but I'm not sure where you're specifically referring to. Thanks!

Reply:
Fixed per @L-Hudson and @bermudezmt.